examples/platforms/nrf52840/radio.c

```diff
@@ -329,7 +329,7 @@ otError otPlatRadioTransmit(otInstance *aInstance, otRadioFrame *aFrame)
     aFrame->mPsdu[-1] = aFrame->mLength;
 
     nrf_drv_radio802154_channel_set(aFrame->mChannel);
-    nrf_drv_radio802154_tx_power_set(aFrame->mPower);
+    nrf_drv_radio802154_tx_power_set(aFrame->mRssi);
```

Comment:
It looks that TX power set before transmission should be `mTxPowerConfig` instead of `mRssi`.